builder/builder_pod.go

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2022 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package builder
+
+import (
+	"context"
+	apiv08 "github.com/davidesalerno/kogito-serverless-operator/api/v08"
+	"github.com/davidesalerno/kogito-serverless-operator/constants"
+	"github.com/ricardozanini/kogito-builder/util/log"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type Buildable struct {
+	client client.Client
+	Ctx    context.Context
+}
+
+func NewBuildable(client client.Client,
+	Ctx context.Context) Buildable {
+	return Buildable{
+		Ctx:    Ctx,
+		client: client,
+	}
+}
+
+func (buildable *Buildable) GetSwfBuild(ctx context.Context, req ctrl.Request, workflowID string, client client.Client) (apiv08.KogitoServerlessBuild, error) {
+	buildInstance := &apiv08.KogitoServerlessBuild{}
+	buildInstance.Spec.SwfName = workflowID
+	error := client.Get(ctx, types.NamespacedName{Namespace: req.NamespacedName.Name, Name: workflowID}, buildInstance)
```

Comment:
Here you are checking if there is a Build CR into the req.NamespacedName.Name namespace and at line 52 you are creating it into another namespace (BUILDER_NAMESPACE_DEFAULT).
 Why?